src/actions/effect/EffectActions/EffectActionWithLevel.ts

```diff
@@ -3,7 +3,7 @@ import LeveledEffectAction from "./LeveledEffectAction";
 /**
  * @class EffectActionWithLevel
  * @augments LeveledEffectAction
- * @description A class for all effects that include a level method
+ * @description A class that provides a built in level() method that sets that level of the effect
```

Comment:
I think this should be "that set the level of the effect"

Reply:
I'm not sure what you mean ,can you write the entire docstring?

Reply:
A class that provides a built in level() method that sets the level of the effect